### DISCUSSION TITLE
Alarm panel card status unknown

### DISCUSSION THREAD

**Discussion by coolstuff99** [Dilbert66/esphome-dsckeybus#117] [Q&A] [answered]
Hi community :)

I recently updated my old version of dsckeybus and realized my basic alarm panel does  not work as before.
Even though I have selected Arm home and Arm away, but non of them appears and always show "DISARM" and Unknown status in the alarm panel as shown in the image:
![imagen](https://github.com/Dilbert66/esphome-dsckeybus/assets/77806850/2abc2dcd-87de-4b65-938d-52f96f8e78ad)

Do anyone know why is this happening? I have 2 physical keyboards. Can this be the issue?

Thank you,

**Comment by Dilbert66** [comment, marked as answer]
Make sure the input names of the yaml fields match what you are using in home assistant.  They have changed in the new sample yaml. You can change them to whatever you need.

**Comment by coolstuff99** [comment]
Ooohh got it.
It was the sensor name "dscalarm_partition_1_status" in the configuration.yaml the one that was not corresponding to the same I have in Home Assistant. Now it works.
Thank you @Dilbert66 for this amazing project!

**Reply by coolstuff99** [reply]
Argg still not fully working.

Now it appears Arm away and Arm stay in the UI panel but I am unable to call any of these services... because these ones don't exist in home assistant. How can I check why are not published as pointed out in the documentation? or I have to do something else?

Thank you,

**Reply by coolstuff99** [reply]
I end up deleting the device + all entities and adding it again. Now services are detected and work as expected.
Thank you!
